tests/test_named_lib.py

```diff
@@ -0,0 +1,21 @@
+from cocotb_test.simulator import run
+import pytest
+import os
+
+tests_dir = os.path.dirname(__file__)
+
+@pytest.mark.skipif(os.getenv("SIM") != "questa", reason="Named libraries only supported for Questa simulator")
```

Comment:
Should this work for GHDL?

Reply:
I just installed GHDL on my system (what a hassle...) and added support for named libraries. Check out the update!